writing-documentation: removed redundant link (#12308)
elixir-lang/elixir: lib/elixir/pages/writing-documentation.md

> Elixir documentation is written using Markdown. There are plenty of guides on Markdown online, we recommend the ones available at GitHub as a getting started point:

Reviewer: ```suggestion
Elixir documentation is written using Markdown. There are plenty of guides on Markdown online, we recommend the one from GitHub as a getting started point:
```
